tests/test-umockdev-record.vala

```diff
@@ -202,11 +202,17 @@ t_system_single ()
     assert_in("E: DEVNAME=/dev/zero", sout);
 #if HAVE_SELINUX
     // we may run on a system without SELinux
+    int res = -1;
     if (FileUtils.test("/sys/fs/selinux", FileTest.EXISTS)) {
         string context;
-        assert_cmpint (Selinux.lgetfilecon ("/dev/null", out context), CompareOperator.GT, 0);
-        assert_in("E: __DEVCONTEXT=" + context + "\n", sout);
-    } else {
+        res = Selinux.lgetfilecon ("/dev/null", out context);
```

Comment:
Do you know in which situations this can fail?

If we do that, then I think the /sys/fs/selinux check is moot, and can be dropped.

Reply:
Thanks for taking a look at this Pull Request!

Looking at the manpage, and reporting here for convenience:

```
If the context does not exist, or the process has no access to this attribute, errno is set to ENODATA.

If extended attributes are not supported by the filesystem, or are dis- abled, errno is set to ENOTSUP.

The errors documented for the [stat](https://linux.die.net/man/2/stat)(2) system call are also applicable here.
```

In my case, indeed it fails with `ENODATA`. In this specific case there is SELinux loaded, set as `permissive` and without any policy installed/loaded. So I think this is what tricked the test case.

Indeed, if we agree in keeping the check for the `lgetfilecon` return value, I think the `/sys/fs/selinux` check can be dropped, which is similar as what umockdev-record is doing.

Reply:
Ack, then let's drop it...